package.json

```diff
@@ -46,10 +35,8 @@
     "eslint-plugin-jest": "^26.0.0",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.28.0",
-    "husky": "^1.3.0",
     "jest": "^27.4.7",
     "lerna": "^4.0.0",
-    "lint-staged": "^9.2.0",
```

Comment:
We were using really old versions, so reinstalling seemed simpler than migrating.